CHAID/CHAID.py

```diff
@@ -133,6 +137,12 @@ def __init__(self, choices=None, split=None, indices=None, node_id=0, parent=Non
         self._members = None
         self.is_terminal = is_terminal
 
+        metadata = dep_v.metadata
+        poss_members = [
+            metadata[k] for k in metadata.keys() if k != -1.0
```

Comment:
If the dep variable was -1, 0, 1, 2, 3 wouldn't keeping -1 in makes sense? Hmm it is an odd use case because if they chose not to answer that's data, but if they weren't given the question than they should be excluded...